src/components/themed/MiniButton.tsx

```diff
@@ -47,14 +48,14 @@ export function MiniButton(props: Props) {
   }
 
   return (
-    <TouchableOpacity disabled={disabled || pending} style={[styles.button, dynamicStyles]} onPress={handlePress}>
+    <EdgeTouchableOpacity disabled={disabled || pending} style={[styles.button, dynamicStyles]} onPress={handlePress}>
```

Comment:
This guy has also incorporated the `usePendingPress` hook.

Reply:
Yeah we really ought to decide on one visual transition and stick to it globally...